src/Akeneo/Pim/Enrichment/Bundle/EventSubscriber/BusinessEvent/DispatchProductCreatedAndUpdatedEventSubscriber.php

```diff
@@ -89,6 +89,20 @@ public function dispatchBufferedProductEvents(): void
 
         try {
             $this->messageBus->dispatch(new BulkEvent($this->events));
+            foreach ($this->events as $event) {
+                $this->logger->info(
+                    json_encode(
+                        [
+                            'type' => $event->getName(),
+                            'uuid' => $event->getUuid(),
+                            'author' => $event->getAuthor()->name(),
+                            'author_type' => $event->getAuthor()->type(),
+                            'timestamp' => $event->getTimestamp(),
+                        ],
```

Comment:
```suggestion
                    json_encode(
                        [
                          'type' => 'event_api.dispatch_event',
                          'event' => [
                                'type' => $event->getName(),
                                'uuid' => $event->getUuid(),
                                'author' => $event->getAuthor()->name(),
                                'author_type' => $event->getAuthor()->type(),
                                'timestamp' => $event->getTimestamp(),
                            ]
                        ],
```
Suggestion on the format